templates/firstbyte/base.html

```diff
@@ -32,6 +32,7 @@
 
         <div id="navbar-links" class="collapse navbar-collapse">
             <div class="navbar-nav">
+                <p class="nav-item nav-link mx-2">FIRSTBYTE</p>
```

Comment:
Should we remove the logo in this PR, what do you think?

Reply:
We probably should, with the color change it's not very noticeable anyways.